src/app/components-small/template-tab-form/template-tab-form.component.ts

```diff
@@ -0,0 +1,53 @@
+import { Component, OnInit, Input, Output, SimpleChanges, EventEmitter } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+
+@Component({
+    selector: 'qhana-template-tab-form',
+    templateUrl: './template-tab-form.component.html',
+    styleUrls: ['./template-tab-form.component.sass']
+})
+export class TemplateTabFormComponent implements OnInit {
+    @Input() formValues: {
```

Comment:
Refactor larger type definitions into interface definitions (and export them if required) to avoid cases where the type specification needs linebreaks.